Gemfile.lock

```diff
@@ -355,9 +355,6 @@ GEM
     html_tokenizer (0.0.7)
     htmlentities (4.3.4)
     http_accept_language (2.1.1)
-    httparty (0.18.1)
-      mime-types (~> 3.0)
-      multi_xml (>= 0.5.2)
```

Comment:
party's over 

![pack it up, the party's over](https://media.giphy.com/media/hveUG4QkNXljkIt96S/giphy.gif)

Reply:
👏🏼 👏🏼